features/diffing.feature

```diff
@@ -25,13 +25,60 @@ Feature: diffing
     Then the output should contain:
       """
              Diff:
-             @@ -1,4 +1,4 @@
+             @@ -1,5 +1,5 @@
               this is the
              -  expected
              +  actual
                   string
       """
 
+  Scenario: diff for a multiline string with trailing newline
+    Given a file named "example_spec.rb" with:
+      """ruby
+      RSpec.describe "a multiline string" do
+        it "is like another string" do
+          expected = "Chewy\nbagel."
+          actual = "Chewy\nbagel.\n"
+          expect(actual).to eq(expected)
+        end
+      end
+      """
+    When I run `rspec example_spec.rb`
+    Then the output should contain:
+      """
+             Diff:
+             @@ -1,3 +1,4 @@
+              Chewy
+              bagel.
+             +
```

Comment:
Actually, this output is what I'm suggesting as the fix :)

From what I understood, the goal was to represent a trailing newline in the diff output instead of nothing at all. This option represents trailing newline diffs the same ways that other newlines diffs in non-trailing situations do. 

I explored adding a string representation where trailing newlines occurred as git does, but it became pretty involved since we add and remove trailing newlines as the output message is crafted in several areas of the code. 

Do you have any ideas of what you'd like to see instead?